tests/end2endtest/es_server.yml

```diff
@@ -0,0 +1,87 @@
+#                                  # Mandatory.
+#
+# server_groups:                   # WBEM server groups.
+#                                  # Optional, default: no groups defined.
+#
+#   NICKNAME42:                    # Nickname of WBEM server group.
+#
+#     - NICKNAME1                  # List of nicknames of servers or server
+#                                  # groups in that group.
+# servers:
+#
+#   NICKNAME1:                     # Nickname of the WBEM server.
+#
+#     description: TEXT            # Short description of the WBEM server.
+#                                  # Mandatory.
+#
+#     contact_name: TEXT           # Name of technical contact for the server.
+#                                  # Optional, default: null.
+#
+#     access_via: TEXT             # Short reminder on the network/firewall/
+#                                  # proxy/vpn used to access the server.
+#                                  # Optional, default: null.
+#
+#     user_defined:
+#
+#       docker_image: ORG/IMAGE:M.N.U
+#                                  # Docker image to pull for setting up the
+#                                  # WBEM server, or null to not pull any
+#                                  # Docker image.
+#                                  # Optional, default: null.
+#
+#       docker_port_mapping:       # Port mapping for Docker image.
+#                                  # Optional, default: straight mapping.
+#
+#         image: PORT              # Port number provided by Docker image.
+#                                  # Mandatory.
+#
+#         local: PORT              # Port number used on local system.
+#                                  # Mandatory.
+#
+#       url: URL                   # URL of the WBEM server; for details see url
+#                                  # parameter of pywbem.WBEMConnection().
+#                                  # Mandatory.
+#
+#       default_namespace: NAMESPACE
+#                                  # Default CIM namespace to use in the
+#                                  # created pywbem.WBEMConnection object.
+#                                  # Optional, default: null.
+#
+#       functions:                 # List of functions implemented by the
+#                                  # server. The function names are specific
+#                                  # to pywbem end2end testing and are checked
+#                                  # by the end2end testcases. If a testcase
+#                                  # requires a particular function and the
+#                                  # server does not implement it, the testcase
+#                                  # is skipped.
+#         - interop                # Has an Interop namespace
+#         - namespaces             # Supports management of namespaces
+#         - profiles               # General support for mgmt profiles
+#         - smis                   # Support for SMI-S profiles
```

Comment:
Not sure if smis is good example.  I could see specific smis profiles however. Some other possibilities include slp,  ssl (i.e. for testing ssl capabilities), subscriptions/indications, pull operations,

Reply:
These functions are not examples, but actually existing end2end tests.

For SMI-S specific tests, we have two test modules test_profile_snia_smis.py that depends on the SMI-.S way of advertising profiles and test_profile_snia_server.py that depends on the SNIA Server profile. So we could split "smis" into two functions, but I think that every SMI-S implementation will support the SNIA Server profile, so you would never find one without the other. Anyway, we could still split "smis" into something like "snia-smis" and "snia-server".

Reply:
DONE - I have split it into "snia-smis" and "snia-server".